src/compiler/tracing.ts

```diff
@@ -1,16 +1,23 @@
 /*@internal*/
 /** Tracing events for the compiler. */
-namespace ts.tracing {
+
+namespace ts {

```

Comment:
What about adding `export const startTracing = tracingEnabled.startTracing;` so `tracingEnabled` use would be limited to this file?

Reply:
Good idea, see if what I pushed now follows what you suggest...

Reply:
The split namespace LGTM.